proto/stride/stakeibc/host_zone.proto

```diff
@@ -44,5 +44,6 @@ message HostZone {
     (gogoproto.nullable) = false
   ];
   string address = 18 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+  bool halted = 19;
```

Comment:
can you increment next id on line 11 to `20`?

Reply:
Ok

Reply:
https://github.com/Stride-Labs/stride/pull/606/commits/3d8770345ab1fe824ba09674e901692039714544